### PR TITLE
formated the graphs in a row of three

### PR DESCRIPTION
**Pull request by richardtully**
Problem: Graphs are formatted poorly, they look ugly in the report.

Solution: Format them with flexboxes

Jira: https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?selectedIssue=AT-86

Next steps: Try to format the graph legends and make the bar chart fit more neatly.

co-authored by Colin

### CODE REVIEW

File: src/components/Dashboard/DashboardELement/DashboardElement.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+const DashboardElement = (props) => {
+    if (props.data.length > 0) {
```
Comment (dearshrewdwit) [inline src/components/Dashboard/DashboardELement/DashboardElement.js:4]: this check shouldn't be necessary - what's the reason for it?

File: src/components/Reports/Reports/Reports.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import React, { Component } from "react";
 import { Doughnut, Bar } from "react-chartjs-2";
-import Report from './Report'
-import ReportHeader from "./ReportHeader";
+import Report from '../Report/Report'
+import ReportHeader from "../ReportHeader/ReportHeader";
 import buildData from "./buildData";
 import jsPDF from "jspdf";
+import './Reports.css'
+// import { options } from "jest-validate/node_modules/@types/yargs";
```
Comment (dearshrewdwit) [inline src/components/Reports/Reports/Reports.js:8]: remove commented code

File: src/components/Reports/Reports/Reports.js [hunk 3]
```diff
@@ -21,7 +25,7 @@ class Reports extends Component {
     }
     componentDidMount() {
         this.exitIframeListener()
-        fetch(`${process.env.REACT_APP_API_URL}/api/cohorts/${this.props.cohortId}/reports`)
+        fetch(`${process.env.REACT_APP_API_URL}/api/cohorts/1/reports`)
```
Comment (dearshrewdwit) [inline src/components/Reports/Reports/Reports.js:28]: Should the ID be hardcoded here?

File: src/components/Reports/Reports/Reports.js [hunk 4]
```diff
@@ -46,18 +50,24 @@ class Reports extends Component {
         }
     );
     }
+    
     render() {
         return(
             <div>
                 <button id="previewPDF" onClick={this.createPdf}>PDF</button>
                 <div id='test2'>
```
Comment (dearshrewdwit) [inline src/components/Reports/Reports/Reports.js:58]: I've seen this changed before - is this change in a different PR?

File: src/components/Dashboard/Dashboard/buildData.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+const buildLabelArray = (data) => data.map(obj => obj.type)
```
Comment (dearshrewdwit) [inline src/components/Dashboard/Dashboard/buildData.js:1]: You could extract these to a utility module, and rename more abstractly

```js
// utils/objectArrays.js
const valuesByProp = (array, prop) => array.map(obj => obj[prop])
const parsedIntsByProp = (array, prop) => array.map(obj => parseInt(obj[prop]))

module.exports {
  valuesByProp: valuesByProp,
  parsedIntsByProp: parsedIntsByProp
}
```

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Good. See comments

File: src/components/Reports/Reports/Reports.js (hunk 4 above)
Comment (b7rch4) [reply src/components/Reports/Reports/Reports.js:57]: I don't think so no. This change is adding a flex-box which we use to format the graphs on the page into a row of three.

Comment (dearshrewdwit) [reply src/components/Reports/Reports/Reports.js:57]: So what's the reason for div with id of 'test2'?

**Comment by dearshrewdwit** [comment]
@richardtully @b7rch4 , you're doubling the commits instead of squashing them ❗

**Comment by richardtully** [comment]
We think it's going to be easier to just redo this branch considering how small our changes are.

**Pull request closed without merging**